cmd/builder/README.md

```diff
@@ -131,6 +131,16 @@ replaces:
   - github.com/open-telemetry/opentelemetry-collector-contrib/internal/common => github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.40.0
 ```
 
+The builder also allows setting the scheme to use as the default URI scheme via `resolver_settings.default_uri_scheme`:
+
+```yaml
+conf_resolver_settings:
+   default_uri_scheme: "env"
+```
+
+This tells the builder to produce a collector that uses the `env` scheme when expanding configuration that does not
+provide a scheme, such as `${HOST}` (instead of doing `${env:HOST}`)
```

Comment:
```suggestion
This tells the builder to produce a Collector that uses the `env` scheme when expanding configuration that does not
provide a scheme, such as `${HOST}` (instead of doing `${env:HOST}`).
```